sdks/python/apache_beam/runners/direct/evaluation_context.py

```diff
@@ -56,6 +56,11 @@ def __init__(self, view):
     self.value = None
     self.has_result = False
 
+  def __repr__(self):
+    elements = ', '.join([
+        str(elm) for elm in self.elements] if self.elements else ['<empty>'])
```

Comment:
<!--new_thread; commit:2053e91ce5cf41b329a4bf8bdb48a06b79b3626b; resolved:0-->
Can you do the following?

```
elements_string = ', '.join(str(elm) for elm in self.elements) if self.elements else '<empty>'
```

Reply:
Done